_posts/en/newsletters/2020-10-28-newsletter.md

```diff
@@ -0,0 +1,233 @@
+---
+title: 'Bitcoin Optech Newsletter #121'
+permalink: /en/newsletters/2020/10/28/
+name: 2020-10-28-newsletter
+slug: 2020-10-28-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes the disclosure of two vulnerabilities
+in LND and includes our regular sections with summaries of popular
+questions and answers from the Bitcoin StackExchange, announcements of
+releases and release candidates, and descriptions of changes to popular
+Bitcoin infrastructure software.
+
+## Action items
+
+- **Ensure LND nodes upgraded to 0.11+:** as warned in [Newsletter
+  #119][news119 lnd], a full disclosure of two vulnerabilities in old
+  versions of LND has been made public.  Both vulnerabilities allow funds
+  to be stolen from an affected LN node, so it is strongly recommended
+  to immediate upgrade to LND 0.11.0-beta or LND 0.11.1-beta.  Other LN
+  implementations are not affected by the described vulnerabilities.
+
+## News
+
+- **Full disclosures of recent LND vulnerabilities:** two
+  vulnerabilities affecting LND were fully disclosed this week with
+  posts to the Lightning-Dev mailing list by both LND co-maintainer
+  Conner Fromknecht and vulnerability discoverer Antoine Riard.
+
+    - **[CVE-2020-26895][] acceptance of non-standard signatures:** in
+      some cases, LND would accept a transaction signature that Bitcoin
+      Core would not relay or mine by default.  When the transaction
+      containing the unrelayable signature fails to confirm, a timelock
+      eventually expires and the attacker is able to steal funds they
+      previously paid to the vulnerable user.
+
+        This is a consequence of the *S* value in Bitcoin ECDSA
+        signatures being equally valid on either the high side or low
+        side of Bitcoin's elliptical curve (which, like all such curves,
+        is symmetric).  Before segwit, inverting an *S*
+        value---something anyone can do to any valid
+        signature[^invert]---would change a transaction's txid, a
+        problem known as *transaction malleability*.  After a major
+        Bitcoin exchange lost funds by not properly handling mutated
+        txids, [Bitcoin Core 0.11.1][] was released with a policy not to
+        relay or mine transactions with *S* values on the high side of
+        the curve---requiring anyone who wanted to exploit this form of
+        malleability to buy or rent mining hashrate.  Even though segwit
```

Comment:
or collude with miners who aren't enforcing this standardness rule. Perhaps that falls under 'renting mining hashrate'